src/__init__.py

```diff
@@ -96,6 +96,8 @@ def get_env_bool( name, default):
 # Unset ascender / descender corrections
 g_skip_quad_corrections = 0
 
+# additional word delmiters
+g_word_delimiters = [0] * 65
```

Comment:
Feels like we should do `g_word_delimiters = list()`. Or, better, pass around as a fn param so things are thread safe.